package.json

```diff
@@ -58,7 +59,9 @@
     "grunt-svgstore": "0.5.0",
     "jsass-vars": "0.0.3",
     "load-grunt-config": "0.16.0",
-    "perfjankie": "1.2.2"
+    "perfjankie": "1.2.2",
+    "promise": "^7.0.3",
+    "selenium-webdriver": "^2.46.1"
```

Comment:
We are already including grunt-selenium server this seems redundant. It also seems like a dependency of `axe-webdriverjs` not of chassis.